src/ports/postgres/modules/deep_learning/madlib_keras.py_in

```diff
@@ -321,9 +321,10 @@ def fit(schema_madlib, source_table, model,model_arch_table,
                   description, aggregate_runtime, class_values])
 
     create_output_table = plpy.prepare("""
-        CREATE TABLE {0} AS
-        SELECT $1 as model_data""".format(model), ["bytea"])
-    plpy.execute(create_output_table, [model_state])
+        CREATE TABLE {0} AS SELECT
+        $1 as model_data,
+        $2 as {1}""".format(model, Format.MODEL_ARCH), ["bytea", "text"])
```

Comment:
I think the type should be JSON rather than TEXT:

$2::JSON as {1} ...

Reply:
you are right it should be json. I will fix it